### ISSUE TITLE
has not support Kingfisher 6.0 ?

### ISSUE THREAD

**Issue by tisfeng** [Yeatse/KingfisherWebP#46]
my Podfile:

```
 pod 'Kingfisher', '~> 6.0'
 pod 'KingfisherWebP'
```

pod install, log error

```
Analyzing dependencies
[!] CocoaPods could not find compatible versions for pod "Kingfisher":
  In snapshot (Podfile.lock):
    Kingfisher (= 6.0.1, ~> 6.0)

  In Podfile:
    Kingfisher (~> 6.0)

    KingfisherWebP was resolved to 0.2.2, which depends on
      Kingfisher (~> 3.0)
```

**Comment by tisfeng** [comment]
maybe a workaround,  modify Podfile:
```
  pod 'Kingfisher', '~> 5.8'
  pod 'KingfisherWebP', '~> 1.0'
```
pod install successfully.

**Comment by Yeatse** [comment]
Resolved in `v1.2.0`
